src/cryptmd5.ts

```diff
@@ -66,6 +66,17 @@ export class Hash {
     constructor(public value: string) { }
 }
 
+
+export function janus_hash(password: string, salt: string): Hash {
+    if('' === password) {
+        return new Hash('');
+    } else {
+        return crypt_md5(password, salt);
+    }
+}
+
+
+
 /**
```

Comment:
> The disadvantage is, that I still have to make the decision about the type on my side before calling changeUser(). Actually I wanted to get rid of that decision...

I understand what you mean here and I'd agree agree with you. But then making such "special cases" explicit at the right place in the code justifies a little redundancy.

Done in f736baa.